packages/statusbar/src/statusbar.ts

```diff
@@ -112,6 +123,13 @@ export class StatusBar extends Widget implements IStatusBar {
     super.dispose();
   }
 
+  private WindowZoomedLevel = () => {
+    // Get the current browser zoom level.
+    const browserZoomLevel = Math.round(window.devicePixelRatio * 100);
+    // Check if the zoom level is greater than or equal to 400%.
+    return browserZoomLevel >= 400;
+  };
```

Comment:
Hmm, I'm not confident that this works. Have this been tested? Are we sure this works?

Reply:
Yeah this has been tested and seems to work how we want it too.

Reply:
I still have concerns about this approach. The intent, as I understand it, is to start hiding things in the status bar at 400% zoom. However, on my Mac's screen, at 100% zoom, `window.devicePixelRatio` is 2, so at 200%, the ratio is 4 and causes the status bar to start hiding stuff. So this creates inconsistent behavior between high pixel density screens versus regular ones. 

Another thing I noticed while testing this, not sure if it's related: when the file browser is open, the status bar does not start hiding items at 400% zoom. I actually first have to hide the status bar, *then* the items will hide at 400%. Does this happen for you too? Any idea what's going on?

Reply:
Hi @gabalafou I have made a change to the status bar zoom functionality that should work no matter your device pixel ration. Please let me know if there is any other changes that need to be made.